ports/hikogui/portfile.cmake

```
@@ -1,25 +1,22 @@
-vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO hikogui/hikogui
-    REF v0.7.0
-    SHA512 64555c67e5a44f336a3528d3a894d43e2751a1f4e4e4d9f6618c085ef0be4a502610a36625d95f79298080cb483f7361a758f7c43b4784999b0b5d839baacb28
+    REF v0.8.1
```

```suggestion
    REF "v${VERSION}"
```
Suggest using new functions